.github/workflows/pythonapp-gpu.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           - "PT17+CUDA110"
           - "PT18+CUDA102"
```

Comment:
I forgot to mention that now there is no test for v1.9 pytorch, please replace at least one of the previous tests with a 1.9 test

Reply:
OK, sure, let me update it.

Thanks for your reminder.

Reply:
I added the PT19 test back and deleted the previous PT18 test with `21.02` docker, I feel maybe it's typo in the previous config:
https://github.com/Project-MONAI/MONAI/blob/dev/.github/workflows/pythonapp-gpu.yml#L42
If not typo, we also covered `21.02` in the crown test.

Thanks.